contrib/babelfishpg_tsql/src/pl_handler.c

```diff
@@ -4053,6 +4053,8 @@ terminate_batch(bool send_error, bool compile_error)
 	bool		error_mapping_failed = false;
 	int			rc;
 
+	HOLD_INTERRUPTS();
```

Comment:
what is the implication of doing HOLD but missing resume part? I think in large enough functions new code can be added which missed doing resume at right places

Reply:
RESUME part is there at the end of the function

Reply:
I meant someone can add new code and return without doing resume. What is the implication of that?